stdlib/public/runtime/Reflection.mm

```diff
@@ -1100,6 +1100,7 @@ static Mirror ObjC_getMirrorForSuperclass(Class sup,
 //// define what these will be.
 /// \returns the demangled name. Returns nullptr if the input String is not a
 /// Swift mangled name.
+#ifndef __FreeBSD__
```

Comment:
Can you explain the rationale behind this change? As far as I can tell you only commented out the demangler API. Thanks for your hard work, btw!

Reply:
Thanks! It took enormous amount of time to locate this piece of code. I found that both `strdup` and `_swift_strlcpy` in this function results to objc being linked. As far as I understood from the comment it's not used anywhere directly. I will file a bug on jira.

Reply:
I found that it's not the functions themselves but the combination with std:string, see my comment in the PR thread.